scenes/ui_elements/shared_components/theme.tres

```diff
@@ -1,23 +1,196 @@
+
+[sub_resource type="StyleBoxTexture" id="StyleBoxTexture_7k42u"]
+content_margin_bottom = 18.0
+texture = ExtResource("1_rtjv3")
+texture_margin_left = 64.0
+texture_margin_right = 64.0
+axis_stretch_horizontal = 2
+axis_stretch_vertical = 2
+
+[sub_resource type="StyleBoxTexture" id="StyleBoxTexture_mu1ca"]
+content_margin_left = 96.0
+content_margin_top = 64.0
+content_margin_right = 96.0
+content_margin_bottom = 64.0
+texture = ExtResource("5_mu1ca")
+texture_margin_left = 64.0
+texture_margin_top = 64.0
+texture_margin_right = 64.0
+texture_margin_bottom = 64.0
+axis_stretch_horizontal = 1
+axis_stretch_vertical = 1
+
+[sub_resource type="StyleBoxTexture" id="StyleBoxTexture_qmeow"]
+content_margin_bottom = 18.0
+texture = ExtResource("7_mtpqe")
+texture_margin_left = 64.0
+texture_margin_right = 64.0
+axis_stretch_horizontal = 2
+axis_stretch_vertical = 2
+
 [resource]
+default_font = ExtResource("1_4jodi")
+default_font_size = 36
+BigFlatButton/base_type = &"FlatButton"
+BigFlatButton/font_sizes/font_size = 60
+Button/colors/font_color = Color(0.12, 0.12, 0.12, 1)
+Button/colors/font_disabled_color = Color(0.12, 0.12, 0.12, 0.501961)
+Button/colors/font_focus_color = Color(0.05, 0.05, 0.05, 1)
+Button/colors/font_hover_color = Color(0.05, 0.05, 0.05, 1)
+Button/colors/font_hover_pressed_color = Color(0, 0, 0, 1)
+Button/colors/font_pressed_color = Color(0, 0, 0, 1)
+Button/styles/disabled = SubResource("StyleBoxTexture_iy6d2")
+Button/styles/focus = SubResource("StyleBoxTexture_si72l")
+Button/styles/hover = SubResource("StyleBoxTexture_rcupm")
+Button/styles/normal = SubResource("StyleBoxTexture_rcupm")
+Button/styles/pressed = SubResource("StyleBoxTexture_w6oc6")
+CheckButton/styles/focus = SubResource("StyleBoxFlat_lerkg")
+CheckButton/styles/hover = SubResource("StyleBoxFlat_lerkg")
+CheckButton/styles/normal = SubResource("StyleBoxEmpty_7k42u")
+CheckButton/styles/pressed = SubResource("StyleBoxEmpty_lerkg")
 FixedSizeLabel/base_type = &"Label"
 FixedSizeLabel/colors/font_color = Color(0, 0, 0, 1)
 FixedSizeLabel/font_sizes/font_size = 36
-FixedSizeLabel/fonts/font = ExtResource("1_cqcnm")
+FixedSizeLabel/fonts/font = ExtResource("1_4jodi")
 FixedSizeLabelContainer/base_type = &"PanelContainer"
 FixedSizeLabelContainer/styles/panel = SubResource("StyleBoxFlat_m7hnk")
+FlatButton/colors/font_hover_color = Color(1, 1, 1, 1)
+FlatButton/styles/focus = SubResource("StyleBoxTexture_lerkg")
+FlatButton/styles/hover = SubResource("StyleBoxTexture_q6hjb")
+FlatButton/styles/hover_pressed = SubResource("StyleBoxTexture_3vcfk")
+FlatButton/styles/normal = SubResource("StyleBoxTexture_bbim3")
+FlatButton/styles/pressed = SubResource("StyleBoxTexture_3vcfk")
+FlatNextButton/base_type = &"FlatButton"
+FlatNextButton/styles/normal = SubResource("StyleBoxTexture_qndbo")
+GridContainer/constants/h_separation = 32
+GridContainer/constants/v_separation = 32
 HintLabel/base_type = &"Label"
+HintLabel/colors/font_color = Color(1, 1, 1, 1)
```

Comment:
This is the only change that isn't just "merge the two files". `HintLabel` inherits from `Label`, which overrides `font_color`.